modules/navier_stokes/src/fvkernels/PINSFVMomentumAdvection.C

```diff
@@ -105,6 +101,9 @@ PINSFVMomentumAdvection::coeffCalculator(const Elem & elem, const ADReal & mu) c
     // Rhie-Chow coefficient for. "neighbor" is the element across the current FaceInfo (fi)
     // face from the Rhie-Chow element
 
+    const auto face_mu = _mu(std::make_tuple(fi, _cd_limiter.get(), true, elem.subdomain_id()));
```

Comment:
If `_mu` is a function of variables, then it must provide a limiter because this code could get invoked
```c++
template <typename OutputType>
ADReal
MooseVariableFV<OutputType>::getInternalFaceValue(const FaceArg & face) const
{
  const FaceInfo * const fi = std::get<0>(face);
  mooseAssert(fi, "The face information must be non-null");
  const Moose::FV::Limiter * const limiter = std::get<1>(face);
  mooseAssert(limiter, "the limiter must be non-null");
  const bool elem_is_upwind = std::get<2>(face);

  const auto elem_value = getElemValue(&fi->elem());
  mooseAssert(fi->neighborPtr(), "We're supposed to be on an internal face.");
  const auto neighbor_value = getElemValue(fi->neighborPtr());
  const auto & upwind_value = elem_is_upwind ? elem_value : neighbor_value;
  const auto & downwind_value = elem_is_upwind ? neighbor_value : elem_value;
  auto upwind_gradient = [this, elem_is_upwind, fi]() -> const ADRealVectorValue & {
    return elem_is_upwind ? adGradSln(&fi->elem()) : adGradSln(fi->neighborPtr());
  };

  return Moose::FV::interpolate(*limiter,
                                upwind_value,
                                downwind_value,
                                limiter->constant() ? nullptr : &upwind_gradient(),
                                *fi,
                                elem_is_upwind);
}
```